### ISSUE TITLE
Does not work on Linux

### ISSUE THREAD

**Issue by StrongSand94191** [Fcornaire/TF.EX#181]
Using Fortrise 4.7.5, no other mods loaded, TF.EX 0.8.2.
Using Kubuntu 23.10
```
[INFO] Ln: 15 Initializing Steam...
[INFO] Ln: 20 Setting up steam app ID to 251470
[INFO] Ln: 603 [RESOURCE] Initializing  resources...
[INFO] Ln: 603 [RESOURCE] Initializing Metadata: TF.EX by DShad 0.8.2 resources...
[ERROR] Ln: 226 System.ComponentModel.Win32Exception (0x80004005): Access denied
  at System.Diagnostics.Process.StartWithShellExecuteEx (System.Diagnostics.ProcessStartInfo startInfo) [0x00102] in <32fc020d7373456995b3d44fa885e01a>:0 
  at System.Diagnostics.Process.Start () [0x00032] in <32fc020d7373456995b3d44fa885e01a>:0 
  at (wrapper remoting-invoke-with-check) System.Diagnostics.Process.Start()
  at System.Diagnostics.Process.Start (System.Diagnostics.ProcessStartInfo startInfo) [0x0001b] in <32fc020d7373456995b3d44fa885e01a>:0 
  at System.Diagnostics.Process.Start (System.String fileName) [0x00006] in <32fc020d7373456995b3d44fa885e01a>:0 
  at TowerFall.TFGame.OpenLog () [0x00012] in <6a4b78a95e324850a35a269a354e702e>:0 
  at TowerFall.TFGame.InnerMain (System.String[] args) [0x0018e] in <6a4b78a95e324850a35a269a354e702e>:0 
  at TowerFall.TFGame.Main (System.String[] args) [0x00325] in <6a4b78a95e324850a35a269a354e702e>:0 
[ERROR] Ln: 227   at System.Diagnostics.Process.StartWithShellExecuteEx (System.Diagnostics.ProcessStartInfo startInfo) [0x00102] in <32fc020d7373456995b3d44fa885e01a>:0 
  at System.Diagnostics.Process.Start () [0x00032] in <32fc020d7373456995b3d44fa885e01a>:0 
  at (wrapper remoting-invoke-with-check) System.Diagnostics.Process.Start()
  at System.Diagnostics.Process.Start (System.Diagnostics.ProcessStartInfo startInfo) [0x0001b] in <32fc020d7373456995b3d44fa885e01a>:0 
  at System.Diagnostics.Process.Start (System.String fileName) [0x00006] in <32fc020d7373456995b3d44fa885e01a>:0 
  at TowerFall.TFGame.OpenLog () [0x00012] in <6a4b78a95e324850a35a269a354e702e>:0 
  at TowerFall.TFGame.InnerMain (System.String[] args) [0x0018e] in <6a4b78a95e324850a35a269a354e702e>:0 
  at TowerFall.TFGame.Main (System.String[] args) [0x00325] in <6a4b78a95e324850a35a269a354e702e>:0 

```

**Comment by Fcornaire** [comment]
Access denied is weird because that's look like not related to the mod itself
But anyway, i don't expect the mod to load on other OS than windows because i made a custom wrapper that was built to be used on Windows.

There might be a way to load the right DLL based on the OS but even with that, the game might desynch like crazy because of how floating value are managed differently based on many factor

In short, won't work and don't expecti it
Might try to at some point

**Comment by Fcornaire** [comment]
Need to ask Terria but i think it's working now, closing
